src/openms/source/ANALYSIS/MAPMATCHING/MapAlignmentAlgorithmKD.cpp

```diff
@@ -102,9 +107,10 @@ void MapAlignmentAlgorithmKD::fitLOWESS()
     Size n = fit_data_[i].size();
     if (n < 50)
     {
-      LOG_WARN << "Warning: Only " << n << " data points for LOWESS fit of map " << i << ". Consider adjusting RT and m/z tolerance, decreasing min_rel_cc_size, or increasing max_nr_conflicts." << endl;
+      LOG_WARN << "Warning: Only " << n << " data points for LOWESS fit of map " << i << ". Consider adjusting RT or m/z tolerance or max_pairwise_log_fc, decreasing min_rel_cc_size, or increasing max_nr_conflicts." << endl;
     }
-    transformations_[i] = new TransformationModelLowess(fit_data_[i], Param());
+    const Param& lowess_param = param_.copy("LOWESS:", true);
+    transformations_[i] = new TransformationModelLowess(fit_data_[i], lowess_param);
```

Comment:
Just wondering, since it does not seem to happen in this class.
Who is responsible for deletion?